booking-engine-api/operations/hotels.md

```diff
@@ -419,11 +423,21 @@ The availability and pricing is returned for each applicable combination of occu
 | `ProductIds` | array of string | optional | Unique identifiers of products which should be included into pricing calculations. |
 | `CurrencyCode` | string | optional | ISO 4217 code of the currency. If specified the prices in response will contain only single currency based on the code provided. [Supported currency codes](../guidelines/supported-currency-codes.md) |
 | `VoucherCode` | string | optional | Voucher code enabling special rate offerings. |
-| `AdultCount` | number | optional | Requested number of adults. If provided together with `ChildCount`, then `RoomOccupancyAvailabilities` will be computed only for that combination instead of all possible. If `RoomCategory` doesn’t support the given values, then the nearest applicable values are found. |
-| `ChildCount` | number | optional | Requested number of children. |
+| `OccupancyData` | array of [Occupancy Data](#occupancy-data) | required | Occupancy numbers for the reservations. |
+| ~~`AdultCount`~~ | ~~number~~ | ~~optional~~ | ~~Requested number of adults. If provided together with `ChildCount`, then `RoomOccupancyAvailabilities` will be computed only for that combination instead of all possible. If `RoomCategory` doesn’t support the given values, then the nearest applicable values are found.~~ Deprecated |
+| ~~`ChildCount`~~ | ~~number~~ | ~~optional~~ | ~~Requested number of children.~~ Deprecated |
 | `CategoryIds` | array of string | optional | Unique identifiers of categories for which should be the availability computed only. If omitted, availability of all categories is returned instead. |
 | `AvailabilityBlockId` | string | optional | Unique identifier of availability block, if present. When provided, only availability for this block is returned. |
 
+
+#### Occupancy Data
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AgeCategoryId` | string | required | Age category identifier. |
```

Comment:
```suggestion
| `AgeCategoryId` | string | required | Age category identifier - see [Where can I get Age Category ID?](../../FAQ/README.md#where-can-i-get-age-category-id). |
```

Reply:
Ah, I just realised that Occupancy Data is already defined in reservations.md. In that case, rather than duplicate it, I think it would be better to reference it from hotels.md, i.e. ```[Occupancy Data](reservations.md#occupancy-data)```. At the same time, the definition in reservations.md should be updated to reflect the changes suggested here, as follows:

#### Occupancy Data

| Property | Type | Contract | Description |
| :-- | :-- | :-- | :-- |
| `AgeCategoryId` | string | required | Age category identifier - see [Where can I get Age Category ID?](../../FAQ/README.md#where-can-i-get-age-category-id). |
| `PersonCount` | number | required | Number of people in the given age category. |

I hope that's clear :-)